getsolus/packages
# keyd: Add at 2.5.0

**Summary**

Add keyd a key remapper for linux

Resolves getsolus/packages#3616

**Test Plan**

- Set up a config for vim movement
- open micro and make sure capslock+vim navigation works

**Checklist**
